core/templates/dev/head/pages/exploration_editor/settings_tab/settings_tab.html

```diff
@@ -186,8 +189,11 @@ <h3>Advanced Features</h3>
                 </span>
               </button>
             </span>
-            <span class="col-lg-8 col-md-8 col-sm-8 help-block" style="font-size: smaller;">
-              Automatic text-to-speech generates audio from your content for learners to listen to. This feature is enabled by default. It is recommended that you disable this feature if creating an exploration whose content consists of multiple languages.
+            <span ng-if="!isAutomaticTextToSpeechEnabled()" class="col-lg-8 col-md-8 col-sm-8 help-block" style="font-size: smaller;">
+              Automatic text-to-speech feature is disabled. It generates audio from your content for learners to listen to.
```

Comment:
Automatic text-to-speech is disabled. It ...

Reply:
Done!